### PR TITLE
Completed the fibonacci program in ruby

### PR DESCRIPTION
**Pull request by thomaspimentel**
Hi @fharookshaik, I added the Fibonacci sequence in the Ruby language. Let me know if any issues, thank you!

### CODE REVIEW

File: ruby_fibonacci.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+def fibonacci(num)
+    if num <= 0
+        puts "Invalid number supplied."
+        return
+    end
+    num_one = 0
+    num_two = 1
+    n = 0 
+    
+    num.times do |i|
```
Comment (fharookshaik) [inline ruby_fibonacci.rb:10]: Codacy found an issue: [Unused block argument - `i`. You can omit the argument if you don't care about it.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6318302)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- ruby_fibonacci.rb  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6318302&bid=20539347)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- ruby_fibonacci.rb  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6318302&bid=20539347)

**Comment by webscaping** [comment]
Apologies! I removed the block argument i. Let me know if that works. 👍

**Comment by fharookshaik** [comment]
Please provide the screenshot of the working code.

**Comment by thomaspimentel** [comment]
> Please provide the screenshot of the working code.

Hey @fharookshaik here's a screenshot of the code working. Let me know if you need anything else from me, thanks!

Screen Shot 2020-10-17 at 12 24 41 AM

**Comment by fharookshaik** [comment]
Appreciated your effort. As for now I'll merge your code. Expecting to develop it more so that it can return at least 100 fibonacci terms.

Thanks for contributing.

**Comment by thomaspimentel** [comment]
Hi @fharookshaik, thanks for merging. I just wanted to post this screenshot to show you the program does in fact return 100 Fibonacci terms. Let me know if there are any other issues, thanks again!


Screen Shot 2020-10-19 at 12 38 47 AM

**Pull request merged**